core/domain/feedback_jobs_one_off_test.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for feedback related jobs"""
+
+from core.domain import feedback_jobs_one_off
+from core.domain import feedback_services
+from core.platform import models
+from core.tests import test_utils
+
+(feedback_models,) = models.Registry.import_models([models.NAMES.feedback])
+taskqueue_services = models.Registry.import_taskqueue_services()
+
+
+class PopulateMessageCountOneOffJobTest(test_utils.GenericTestBase):
+    """Tests for PopulateMessageCountOneOffJob."""
+
+    def setUp(self):
```

Comment:
Oops forgot to mention last time, sorry! Can we also have a thread with no message, and make sure that the count is 0?

Reply:
Yes, will add it !